Sources/GRPCReflectionService/Model/reflection.proto

```diff
@@ -0,0 +1,141 @@
+// Copyright 2023 The gRPC Authors
```

Comment:
This license header should be unchanged from https://github.com/grpc/grpc-proto/blob/master/grpc/reflection/v1/reflection.proto